class CategoryNotFoundError(Exception):
    pass


class CategoryAlreadyExistsError(Exception):
    pass
